nsv13/code/modules/power/stormdrive.dm

```diff
@@ -199,18 +195,28 @@ Takes  plasma and outputs superheated plasma and a shitload of radiation.
 	if(!in_range(src, usr) || !reactor)
 		return
 	if(href_list["rods_1"])
-		reactor.control_rod_state = RODS_RAISED
+		reactor.control_rod_percent = 0
 		message_admins("[key_name(usr)] has fully raised reactor control rods in [get_area(usr)] [ADMIN_JMP(usr)]")
 		reactor.update_icon()
 	if(href_list["rods_2"])
-		reactor.control_rod_state = RODS_HALFRAISED
+		reactor.control_rod_percent = 33.42
 		reactor.update_icon()
 	if(href_list["rods_3"])
-		reactor.control_rod_state = RODS_HALFLOWERED
+		reactor.control_rod_percent = 23.49
```

Comment:
These numbers are backwards: according to the control_rod_percent formula on line 445, position 2 will now have a target heat of 100, and position 3 will have a target heat of 200. Because position 1 is the maximum heat position and position 4 is the maximum cooling position, this means that the heat level of the control rod presets would be 1>3>2>4.

Reply:
resolved as outdated